pkg/apis/eventing/v1alpha1/trigger_types.go

```diff
@@ -94,6 +112,25 @@ type TriggerStatus struct {
 
 	// SubscriberURI is the resolved URI of the receiver for this Trigger.
 	SubscriberURI string `json:"subscriberURI,omitempty"`
+
+	Address duckv1alpha1.Addressable `json:"address,omitempty"`
+
+	Type TriggerType `json:"type,omitempty"`
```

Comment:
`Type` is vague and possibly confused with CloudEvents type. Can we call this something else like `DeliveryMode`?